PusherMainView/PusherMainView/Base.lproj/Pusher.storyboard

```diff
@@ -263,18 +263,38 @@
-                                        <buttonCell key="cell" type="check" title="Sandbox" bezelStyle="regularSquare" imagePosition="left" inset="2" id="wrh-SG-FTA">
-                                            <behavior key="behavior" changeContents="YES" doesNotDimImage="YES" lightByContents="YES"/>
-                                            <font key="font" metaFont="system"/>
-                                        </buttonCell>
-                                    </button>
+                                    <stackView distribution="fill" orientation="horizontal" alignment="top" spacing="10" horizontalStackHuggingPriority="249.99998474121094" verticalStackHuggingPriority="249.99998474121094" detachesHiddenViews="YES" translatesAutoresizingMaskIntoConstraints="NO" id="f27-o8-ZxL">
+                                        <rect key="frame" x="158" y="68" width="185" height="16"/>
+                                        <subviews>
+                                            <button verticalHuggingPriority="750" translatesAutoresizingMaskIntoConstraints="NO" id="blG-ra-9fI">
+                                                <rect key="frame" x="-2" y="-1" width="79" height="18"/>
+                                                <buttonCell key="cell" type="check" title="Sandbox" bezelStyle="regularSquare" imagePosition="left" inset="2" id="wrh-SG-FTA">
+                                                    <behavior key="behavior" changeContents="YES" doesNotDimImage="YES" lightByContents="YES"/>
+                                                    <font key="font" metaFont="system"/>
+                                                </buttonCell>
+                                            </button>
+                                            <button verticalHuggingPriority="750" translatesAutoresizingMaskIntoConstraints="NO" id="e4Z-03-8g1">
+                                                <rect key="frame" x="85" y="-1" width="100" height="18"/>
+                                                <buttonCell key="cell" type="check" title="Live Activity" bezelStyle="regularSquare" imagePosition="left" inset="2" id="tT9-Se-2Zs">
+                                                    <behavior key="behavior" changeContents="YES" doesNotDimImage="YES" lightByContents="YES"/>
+                                                    <font key="font" metaFont="system"/>
+                                                </buttonCell>
+                                            </button>
+                                        </subviews>
+                                        <visibilityPriorities>
+                                            <integer value="1000"/>
+                                            <integer value="1000"/>
+                                        </visibilityPriorities>
+                                        <customSpacing>
+                                            <real value="3.4028234663852886e+38"/>
```

Comment:
Hmm... are those intended values? @malayli

Reply:
@imaginaris We already have this number (issue?) in all the other stack views on master branch:
https://github.com/rakutentech/macos-push-tester/blob/master/PusherMainView/PusherMainView/Base.lproj/Pusher.storyboard#L316-L327

Therefore I do not know if it is an issue. It seems to be automatically added.

Reply:
Ok. I just wanted to be sure. It's fine then